### PR TITLE
Pipeline 2 chef changes

### PR DESCRIPTION
**Pull request by anguss00**
Final PR to incorporate pipeline 2 required chef changes

### CODE REVIEW

File: cookbooks/imos_po/attributes/pipeline.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+default['imos_po']['logging']['directories'] = %w(harvest watchservice process celery)
+
+# error directories
+default['imos_po']['error']['root'] = '/mnt/ebs/error'
+
+# lock file location
+default['imos_po']['pipeline']['lock_file_dir'] = '/var/run/pipeline'
+
+# TODO: fix this for production, OK for po box
+default['imos_po']['pipeline']['git_ssh'] = File.join('home', 'vagrant', '.ssh', 'wrappers', 'git_deploy_wrapper.sh')
```
Comment (lwgordonimos) [inline cookbooks/imos_po/attributes/pipeline.rb]: Can this be removed now?

File: cookbooks/imos_po/attributes/pipeline.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# Celery configuration settings - the config files are included in the pipeline package,
+# this is just where chef expects them to be
+default['imos_po']['pipeline']['celeryd']['backend'] = 'rabbitmq'
```
Comment (lwgordonimos) [inline cookbooks/imos_po/attributes/pipeline.rb]: This isn't used, so maybe remove?

File: cookbooks/imos_po/recipes/data_services.rb [hunk 3]
```diff
@@ -109,6 +109,7 @@ def imos_po_credentials
   'ERROR_DIR'             => node['imos_po']['data_services']['error_dir'],
   'ARCHIVE_IMOS_DIR'      => node['imos_po']['data_services']['archive_dir'],
   'WIP_DIR'               => node['imos_po']['data_services']['wip_dir'],
+  'PROCESSING_DIR'        => node['imos_po']['data_services']['wip_dir'],
```
Comment (lwgordonimos) [inline cookbooks/imos_po/recipes/data_services.rb]: Should be 'processing_dir'

File: Berksfile.lock [hunk 4]
```diff
@@ -365,7 +365,7 @@ GRAPH
   squid (3.1.0)
   ssl (1.1.1)
   sudo (3.3.1)
-  supervisor (0.4.12)
+  supervisor (0.4.2)
```
Comment (lwgordonimos) [inline Berksfile.lock]: This makes me nervous... what was the reason for this? Has the potential to break existing pipelines...

Comment (anguss00) [reply Berksfile.lock]: Ok, dredged it https://github.com/poise/supervisor/issues/43.

Comment (anguss00) [reply Berksfile.lock]: Fixed in master, but master of poise-supervisor has commits that are not in 0.4.12

**Comment by anguss00** [comment]
I think we're good to go from my end @lwgordonimos

**Pull request merged**